scripts/rebase-ledger.js

```diff
@@ -0,0 +1,48 @@
+const fs = require("fs");
+const sc = require("../node_modules/sourcecred/dist/server/api.js").sourcecred;
+const isEqual = require("../node_modules/lodash.isequal");
```

Comment:
This dependency isn't explicitly in the `package.json`, and it should be.
We're making the assumption sourcecred will pull it in, so if they removed it this script would fail.

Reply:
accepted